src/main/resources/edu/cornell/kfs/module/purap/cu-ojb-purap.xml

```diff
@@ -3265,10 +3265,10 @@
     <field-descriptor name="preqBankCountryCode" column="PREQ_BNK_CNTRY_CD" jdbc-type="VARCHAR" />
     <field-descriptor name="preqAttentionLineText" column="PREQ_ATTN_LN_TXT" jdbc-type="VARCHAR" />
     <field-descriptor name="preqAdditionalWireText" column="PREQ_ADDL_WIRE_TXT" jdbc-type="VARCHAR" />
-    <field-descriptor name="preqPayeeAccountNumber" column="PREQ_PAYEE_ACCT_NBR" jdbc-type="VARCHAR" conversion="org.kuali.rice.core.framework.persistence.ojb.conversion.OjbKualiEncryptDecryptFieldConversion"/>
+    <field-descriptor name="preqPayeeAccountNumber" column="PREQ_PAYEE_ACCT_NBR" jdbc-type="VARCHAR" conversion="org.kuali.kfs.core.framework.persistence.ojb.conversion.OjbKualiEncryptDecryptFieldConversion"/>
```

Comment:
Short term, I added cynergy-customizations as a test dependency in the pom, overrode the encryptionService with ours and set the encryption key in our properties file. This made the tests pass. Long term, we will need to put cynergy back.